src/extensions/exception-autocapture/index.ts

```diff
@@ -117,7 +118,18 @@ export class ExceptionObserver {
         this.startIfEnabled()
     }
 
-    captureException(errorProperties: Properties) {
+    captureException(error: Error, additionalProperties: Properties = {}) {
+        if (!window || !this.isEnabled || this.isCapturing) {
+            logger.error(LOGGER_PREFIX + ' error tracking is not enabled - cannot capture exception')
```

Comment:
I think the `!window` has to be a different logging message so as not to mislead customers or us when debugging things